package.json

```diff
@@ -101,7 +101,6 @@
     "@angular/cli": "8.0.0",
     "@angular/compiler-cli": "8.0.0",
     "@angular/language-service": "8.0.0",
-    "@compodoc/compodoc": "1.1.9",
```

Comment:
Was compodoc doc causing issues? Just curious over its removal

Reply:
Compodoc was running into memory issues, and my thinking is if we settle on using it we'll dockerize the build tooling for it separately since its a fairly large package.